app/validation/domain.py

```diff
@@ -65,14 +65,14 @@ def validate_to_from_not_equal(self, data):
             raise ValidationError("msg_to and msg_from fields can not be the same.")
 
     # TODO: Validate UUID with mock party
-    @validates('msg_to')
+    @validates("msg_to")
     def validate_to(self, msg_to):
         for item in msg_to:
             self.validate_non_zero_field_length("msg_to", len(item), constants.MAX_TO_LEN)
             if msg_to != 'BRES' and not User.is_valid_user(item):
                 raise ValidationError("{0} is not a valid user.".format(item))
```

Comment:
Should this be consistent with the other ValidationError lines in regards to single quotes rather than double quotes?